client/my-sites/stats/stats-card-upsell/index.tsx

```diff
@@ -1,60 +1,32 @@
 import { translate } from 'i18n-calypso';
```

Comment:
Full file moved to: client/my-sites/stats/stats-card-upsell/stats-card-upsell-wpcom.tsx